components/SortableTable/index.vue

```diff
@@ -662,48 +838,73 @@ export default {
               the value of :class changes. -->
-              <tr :key="get(row,keyField)" class="main-row" :class="{ 'has-sub-row': showSubRow(row, keyField)}" :data-node-id="get(row,keyField)" :data-cant-run-bulk-action-of-interest="actionOfInterest && !canRunBulkActionOfInterest(row)">
+              <tr :key="row.key" class="main-row" :class="{ 'has-sub-row': row.showSubRow}" :data-node-id="row.key" :data-cant-run-bulk-action-of-interest="actionOfInterest && !row.canRunBulkActionOfInterest">
                 <td v-if="tableActions" class="row-check" align="middle">
-                  {{ row.mainRowKey }}<Checkbox class="selection-checkbox" :data-node-id="get(row,keyField)" :value="selectedRows.includes(row)" />
+                  {{ row.mainRowKey }}<Checkbox class="selection-checkbox" :data-node-id="row.key" :value="selectedRows.includes(row.row)" />
                 </td>
                 <td v-if="subExpandColumn" class="row-expand" align="middle">
-                  <i data-title="Toggle Expand" :class="{icon: true, 'icon-chevron-right': true, 'icon-chevron-down': !!expanded[get(row, keyField)]}" @click.stop="toggleExpand(row)" />
+                  <i data-title="Toggle Expand" :class="{icon: true, 'icon-chevron-right': true, 'icon-chevron-down': !!expanded[row.key]}" @click.stop="toggleExpand(row.row)" />
                 </td>
-                <template v-for="col in columns">
+                <template v-for="col in row.columns">
                   <slot
-                    :name="'col:' + col.name"
-                    :row="row"
-                    :col="col"
+                    :name="'col:' + col.col.name"
+                    :row="row.row"
+                    :col="col.col"
                     :dt="dt"
                     :expanded="expanded"
-                    :rowKey="get(row,keyField)"
+                    :rowKey="row.key"
                   >
                     <td
-                      :key="col.name"
-                      :data-title="labelFor(col)"
-                      :align="col.align || 'left'"
-                      :class="{['col-'+dasherize(col.formatter||'')]: !!col.formatter, [col.breakpoint]: !!col.breakpoint, ['skip-select']: col.skipSelect}"
-                      :width="col.width"
+                      :key="col.col.name"
+                      :data-title="col.col.label"
+                      :align="col.col.align || 'left'"
+                      :class="{['col-'+col.dasherize]: !!col.col.formatter, [col.col.breakpoint]: !!col.col.breakpoint, ['skip-select']: col.col.skipSelect}"
+                      :width="col.col.width"
                     >
-                      <slot :name="'cell:' + col.name" :row="row" :col="col" :value="valueFor(row,col)">
+                      <slot :name="'cell:' + col.name" :row="row" :col="col" :value="col.value">
+                        <span v-if="col.formatter && col.formatter === 'LinkDetail'">
```

Comment:
Should this be removed? In theory it should be col.col.formatter... however that breaks as most don't have a `url`, it ignores the `to()` calc from `LinkDetail` and the `value` is a col.col.value which is a function. 
I was testing  with the Machine Pools page for an imported / rke1 cluster

Reply:
Yes it should - this was testing inline a formatter - not needed